### PR TITLE
Add github CI config to deploy to PyPI

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      # make sure we have version info
+      - run: git fetch --tags
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```
Comment (rlopezcoto) [inline .github/workflows/deploy.yml:20]: why 3.8? in lstchain we are still requiring 3.7

**Review by rlopezcoto — COMMENTED** [review]
Let's also follow the same you guys did in protozfits and make a new major release (`v0.8`) for the upload to pypi

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (maxnoe) [reply .github/workflows/deploy.yml:20]: Doesn't matter at all, since all it does is creating the tar balls.

Comment (maxnoe) [reply .github/workflows/deploy.yml:20]: >  in lstchain we are still requiring 3.7

Because protozfits only worked with 3.7, now you could change that ;)

**Comment by maxnoe** [comment]
@rlopezcoto fine with me

**Review by rlopezcoto — APPROVED** [review]

**Pull request merged**